spaghetti/network.py

```diff
@@ -8,7 +8,7 @@
 from libpysal import cg, examples, weights
 try:
     from libpysal import open
-except:
+except BaseException:
```

Comment:
This should be `ImportError`, not `BaseException`?

Reply:
Yes, I ran `autopep8` and then reviewed manually, but this one (and then one below) escaped my attention.

Reply:
@ljwolf 
Before it was mangled I had it like this:
```python
try:
    from libpysal import open
except:
    import libpysal
    open = libpysal.io.open
```

We don't need to raise an ImportError if it's just a try/except, do we?

Reply:
Here, again, we're only interested in the `ImportError`, not any other kind of error that `import libpysal ; open = libpysal.io.open` might raise. So, if we use `except ImportError`, *only errors* matching that pattern get caught. Anything else will go immediately through.